* generate .bundle files, not .so
* set allow_nonpic_in_shlib such that perl modules can be built

--- src/makefiles/Makefile.darwin
+++ src/makefiles/Makefile.darwin
@@ -1,7 +1,8 @@
 AROPT = crs
 AWK= awk
+allow_nonpic_in_shlib = yes
  
-DLSUFFIX = .so
+DLSUFFIX = .bundle
 CFLAGS_SL =
 
 ifdef PGXS
@@ -12,7 +13,7 @@
 
 # Rule for building shared libs (currently used only for regression test
 # shlib ... should go away, since this is not really enough knowledge)
-%.so: %.o
+%.bundle: %.o
 	$(CC) $(CFLAGS) -bundle -o $@ $< $(BE_DLLLIBS)
 
 sqlmansect = 7
